src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs

```diff
@@ -8,7 +8,7 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// <summary>
 /// <see cref="UIView"/> for <see cref="Alert"/>
 /// </summary>
-public class AlertView : UIView
+public sealed class AlertView : UIView
```

Comment:
In `Maui/Forms` isn't common to have don't allow inheritance on controls, do you have a good motivation for that?

Reply:
Inspired by https://www.meziantou.net/performance-benefits-of-sealed-class.htm

Reply:
Yeah, I'm very cautious with inheritance as well. But in this case, we should allow it and we're prepared for it.